book/src/advanced_metrics.md

```diff
@@ -48,3 +48,36 @@ Check to ensure that the metrics are available on the default port:
 ```bash
 curl localhost:5064/metrics
 ```
+
+## Remote Monitoring
```

Comment:
LG, but what about adding some information about `--validator-monitor-auto` and how it works and relates to `--monitoring-endpoint`?

Reply:
Entirely unrelated

Reply:
I'll add a quick note